codespell_lib/data/dictionary.txt

```diff
@@ -28519,6 +28519,8 @@ vulerable->vulnerable
 vulerabuilities->vulnerabilities
 vulerabuility->vulnerability
 vulerabule->vulnerable
+vulernabilities->vulnerabilities
+vulernability->vulnerability
```

Comment:
Would you mind adding this variant of that typo too please:
`vulernable->vulnerable`

Reply:
Absolutely!